### DISCUSSION TITLE
Question on copyright

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#614] [Answers & Questions] [answered]
I would like to make this patch:    https://cables.gl/p/wnyGtL      public and Public domain (when it will be finalised), but I am concerned about "losing" copyright protection on the images in the patch .
any thoughts/ suggestions on this topic?

**Comment by steam0r** [comment, marked as answer]
we offer a variety of licences (all CC) when publishing patches. we will mention the licence the author has chosen on the patch page, when cloning the patch and in several other places.

there is no way for us to enforce these licences somehow, especially not including assets used in the patch. it's also pretty hard for us to know when and how you can "lose copyright" in your legislature and what rights are attached to these assets in the first place.

the best way would be to pick a cc licence that fits best or keep the patch "all rights reserved". this will make your patch publicly available for remixing/cloning and viewing in the editor and ask them to adhere to the licence when cloning.

another option would be to set your patch to "unlisted" (so it does not appear in any top-lists) or "unlisted view-only", so people can only view the patchpage but not open the editor. this also depends on your definition of "public", though...

all of that being said, be aware that you share/distribute these images anyhow, once you export the patch and put it on some website.
